backend-server/src/api/controllers/yeet.ts

```diff
@@ -0,0 +1,201 @@
+import {
```

Comment:
I assume `yeet.ts` was a WIP file? Can we remove it or does something depend on it?